app/scripts/models/application-settings.coffee

```diff
@@ -22,6 +23,7 @@ define [
   class ApplicationSettingsModel extends BaseRelationalModel
 
     initialize: ->
+      @set('fielddbApplication' , FieldDB.FieldDBObject.application)
       @listenTo Backbone, 'authenticate:login', @authenticate
```

Comment:
In app/scripts/collections/forms.coffee you removed the `pouchname` from `"#{url}/#{pouchname}/_design/pages/_view/datums_chronological"`. Why did you do that? Is it because the `url` returned by the commonjs fielddb includes the pouchname of the active corpus?

Also, you're accessing the `fiedldbApplication` as an attribute and not a BB model attribute (i.e., via `.get`), which is what you need to do.